nixos/modules/services/hardware/powerstation.nix

```diff
@@ -0,0 +1,35 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.powerstation;
+in
+{
+  options.services.powerstation = {
+    enable = lib.mkEnableOption "PowerStation";
+    package = lib.mkPackageOption pkgs "powerstation" { };
+  };
```

Comment:
Is there a UI for this already or is the user expected to set dbus stuff manually/scripted?

Reply:
I have opened a PR for the OpenGamepadUI derivation here: https://github.com/NixOS/nixpkgs/pull/368017